FirebaseCore/Sources/Public/FirebaseCore/FIRVersion.h

```diff
@@ -16,9 +16,12 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
 /**
  * This class provides internal access to the Firebase version for any Firebase pod.
  */
+NS_SWIFT_NAME(FirebaseVersion)
```

Comment:
Since we're now exposing this we'll need to make the `init` unavailable, otherwise it will be surfaced in Swift.